commands/mr/view/mr_view.go

```diff
@@ -155,6 +155,31 @@ func printTTYMRPreview(out io.Writer, mr *gitlab.MergeRequest, notes []*gitlab.N
 	if mr.State == "closed" {
 		fmt.Fprintf(out, "Closed By: %s %s\n", mr.ClosedBy.Username, mrTimeAgo)
 	}
+	if mr.Pipeline != nil {
+		fmt.Fprint(out, utils.Bold("Pipeline Status: "))
+		var status string
+		switch s := mr.Pipeline.Status; s {
+		case "failed":
+			status = utils.Red(s)
```

Comment:
We had a change for ci status, that we printed pipeline status yellow, if it's allowed to fail 
like here https://github.com/profclems/glab/blob/8f5df7f1d0fbdfb9eb49d3c90af910c2db5668be/commands/ci/status/status.go#L84-L89

I would actually suggest that we bring out this pipeline status printing out, it's own helper function, so we can centralize it everywhere.

Reply:
Yeah you're right. I will do that